packages/webapp/components/layouts/AccountLayout/Profile/VerifiedCompanyBadge/VerifiedCompanyBadgeSection.tsx

```diff
@@ -0,0 +1,233 @@
+                </Typography>
+                <Typography
+                  type={TypographyType.Footnote}
+                  color={TypographyColor.Tertiary}
+                >
+                  {userCompany.email}
+                  <Typography
+                    className="ml-1"
+                    color={TypographyColor.StatusSuccess}
+                    tag={TypographyTag.Span}
+                  >
+                    Verified
+                  </Typography>
+                </Typography>
+              </div>
+              <Button
+                type="button"
+                variant={ButtonVariant.Tertiary}
+                onClick={() => onDeleteUserCompany(userCompany.email)}
+                icon={<TrashIcon />}
+              />
+            </li>
+          ))}
+        </ul>
+      </div>
+    );
+  };
+
+  const UserHasCompanyInReview = () => {
+    return (
+      <div className="mt-6 max-w-sm">
+        <p className="text-text-tertiary typo-footnote">
+          {userCompanies[0].email}{' '}
+          <span className="ml-1 text-accent-bun-default">In review</span>
+        </p>
+        <Alert
+          className="mt-6"
+          type={AlertType.Success}
+          flexDirection="flex-row"
+        >
+          <AlertParagraph className="!mt-0 flex-1">
+            Your work email has been verified. We’re currently reviewing your
+            company details. Your badge will be added once the review is
+            complete, and you’ll be notified when it’s live.
+          </AlertParagraph>
+        </Alert>
+      </div>
+    );
+  };
+
+  return (
+    <AccountContentSection
+      title="Verified company badge"
+      description={
+        !userCompanyVerified &&
+        'Verify your work email and get a verified company badge on your profile. We won’t require any ID or personal information, just a verification code to complete the process.'
+      }
+    >
+      {userCompanyVerified && <UserHasVerifiedCompanies />}
+      {userCompanyInReview && <UserHasCompanyInReview />}
+      {!userCompanyVerified && !userCompanyInReview && (
+        <>
+          <AccountTextField
+            leftIcon={<MailIcon />}
+            label="Work email"
+            inputId="workmail"
+            name="workmail"
+            placeholder="Work email"
+            value={workEmail}
+            valueChanged={setWorkEmail}
+            type="email"
+            valid={!submitWorkEmailHint}
+            onChange={() => submitWorkEmailHint && setSubmitWorkEmailHint('')}
+            actionButton={
+              <Button
+                size={ButtonSize.Small}
+                variant={ButtonVariant.Primary}
+                icon={<ArrowIcon className="rotate-90" />}
+                type="button"
+                loading={isLoading}
+                disabled={!workEmail && !!submitWorkEmailHint}
+                onClick={() => onSubmitWorkEmail(workEmail)}
+              />
+            }
+          />
```

Comment:
Can we add html input validation, to ensure it's valid email?
I got no errors when I entered this
![image](https://github.com/user-attachments/assets/f72abb12-242b-4e18-a871-d49edfd267ca)

Reply:
Good catch! We also lack email verification on API

Reply:
Yeah, generally lack quite a bit of input validation in the api :sadcat:

Reply:
I did think about it, but then thought people should just format the right thing anyway from the email.
We do check code validation already.

Up to you guys if you think it's needed we can add it.

Reply:
https://github.com/dailydotdev/daily-api/pull/2177

Reply:
In addition to API, we can set email input type to be email, as we should get browser native validation too

Reply:
Type is email already :)

Doesn't work because it's not required and a seperate form basically.
If it was it's own page 🤦 it would be possible.

Reply:
Ahh... :triggered: